### PR TITLE
GEOMESA-1088 Make ImageIO-Ext workswith

### PR DESCRIPTION
**Pull request by AndrewAnnex**
* marked imageio-ext jars provided
* added download scripts to tools
* updated copyright headers in tools shell scripts

Signed-off-by: Andrew Annex <aannex@ccri.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -424,6 +446,7 @@
                 <groupId>org.geotools</groupId>
```
Comment (elahrvivaz) [inline pom.xml]: could you put a comment here that this is 'works with' and so has to stay provided scope?

Comment (AndrewAnnex) [inline pom.xml]: done, put it a line above the groupId

File: pom.xml [hunk 2]
```diff
@@ -421,9 +443,11 @@
                 </exclusions>
             </dependency>
             <dependency>
+                <!-- gt-imageio-ext-gdal is 'works with' so it must stay in provided scope -->
                 <groupId>org.geotools</groupId>
                 <artifactId>gt-imageio-ext-gdal</artifactId>
                 <version>${gt.version}</version>
+                <scope>provided</scope>
```
Comment (elahrvivaz) [inline pom.xml]: @AndrewAnnex Actually, this dep is approved for compile time - it's just the it.geosolutions.imageio-ext stuff we can't use...

File: geomesa-tools/pom.xml [hunk 3]
```diff
@@ -125,6 +125,10 @@
             <artifactId>jcommander</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+        </dependency>
+        <dependency>
```
Comment (jahhulbert-ccri) [inline geomesa-tools/pom.xml]: is this needed in the tools? Did it not work before?

File: pom.xml (hunk 2 above)
Comment (AndrewAnnex) [inline pom.xml]: this had transitive dependencies on the imageio-ext things, and this made it all simpler.

File: geomesa-tools/pom.xml (hunk 3 above)
Comment (AndrewAnnex) [inline geomesa-tools/pom.xml]: @elahrvivaz asked for this to be added

Comment (elahrvivaz) [inline geomesa-tools/pom.xml]: it was transitively included before - we had to explicitly include it after the changes.

Comment (jahhulbert-ccri) [inline geomesa-tools/pom.xml]: ah.

**Comment by elahrvivaz** [comment]
0921d1656a6f388209d1b00afbd54658731489ee

**Pull request closed without merging**